solidity/test/ConverterFactory.js

```diff
@@ -4,13 +4,19 @@ const { expect } = require('../../chai-local');
 
 const ContractRegistry = contract.fromArtifact('ContractRegistry');
 const ConverterFactory = contract.fromArtifact('TestConverterFactory');
-const LiquidTokenConverterFactory = contract.fromArtifact('LiquidTokenConverterFactory');
 const LiquidityPoolV1ConverterFactory = contract.fromArtifact('LiquidityPoolV1ConverterFactory');
 const StandardPoolConverterFactory = contract.fromArtifact('StandardPoolConverterFactory');
+const FixedRatePoolConverterFactory = contract.fromArtifact('FixedRatePoolConverterFactory');
 const TypedConverterAnchorFactory = contract.fromArtifact('TestTypedConverterAnchorFactory');
 const ConverterBase = contract.fromArtifact('ConverterBase');
 const DSToken = contract.fromArtifact('DSToken');
 
+const Factories = {
+    LiquidityPoolV1ConverterFactory,
+    StandardPoolConverterFactory,
+    FixedRatePoolConverterFactory
```

Comment:
fixed rate converter shouldn't be here

Reply:
This is the only place in our code which utilizes and tests contract `FixedRatePoolConverterFactory`.

Reply:
Ok, fair enough